### DISCUSSION TITLE
Linking to an attachment inside a note

### DISCUSSION THREAD

**Discussion by Br0kenSilos** [TriliumNext/Trilium#4561] [Q&A]
Love Trilium and live in it daily. I'm more of a sys admin and not a developer.  Currently running 62.4 on server and desktop client. Although I usually access via server / web. I have a note in which I have attached a file (.msg file from Outlook but assume it could be any kind of file). The attachment is there and works fine. But I want to take some text within the note and link it to that attached file so I can easily open the file or at least see the file referenced. For the life of me I cannot figure out how to do that. I looked at the wiki and it states "A note can own one or more attachments. The attachment is either an image or file which can be displayed/linked in the note which owns it." But doesn't really say how that "link" is created. Ctrl+L doesn't list the attachments. What am I missing?

**Comment by meichthys** [comment]
normally you would be able to click the attachment, then click the vertical three dot menu on the top left to choose "Copy link to clipboard", but this is currently broken since this menu item currently copies the entire attachment to the clipboard instead of the link: #4388

**Reply by Br0kenSilos** [reply]
Yeah. I have noticed that if I try to use "Copy link to clipboard" that it seems to do nothing and the contents of my clipboard are unchanged.

**Reply by Br0kenSilos** [reply]
Something I just noticed... The linking seems to work fine in the Desktop client (Windows in my case). But not in the web client. I may create some anonymized data to test with and add to your bug report.

**Comment by grosserfrank** [comment]
Write @name_of_attachment
